pom.xml

```diff
@@ -527,7 +532,8 @@
             <dependency>
```

Comment:
I used this library time ago, I'm not sure if it's unused now, let me google how to detect unused pom dependencies.